commons/all/src/main/java/org/infinispan/commons/configuration/ClassAllowList.java

```diff
@@ -24,23 +24,27 @@
 import org.infinispan.commons.logging.LogFactory;
 
 /**
- * The {@link ClassWhiteList} maintains classes definitions either by name or regular expression and is used for
+ * The {@link ClassAllowList} maintains classes definitions either by name or regular expression and is used for
  * permissioning.
  * <p>
- * By default it includes regular expressions from the system property "infinispan.deserialization.whitelist.regexps"
- * and fully qualified class names from "infinispan.deserialization.whitelist.classes".
+ * By default it includes regular expressions from the system property "infinispan.deserialization.allowlist.regexps"
+ * and fully qualified class names from "infinispan.deserialization.allowlist.classes".
  * <p>
  * Classes are checked first against the set of class names, and in case not present each of the regular expressions are
  * evaluated in the order supplied.
  *
  * @since 9.4
  */
-public final class ClassWhiteList {
```

Comment:
As this is a public package, I think we need to keep `ClassWhiteList` until 14. We can just make it  extend `ClassAllowList`.

Reply:
Done